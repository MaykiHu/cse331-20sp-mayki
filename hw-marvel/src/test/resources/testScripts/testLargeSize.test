#Tests for the size of a large dataset, the number of nodes, from marvel.tsv
#marvel.tsv should be a valid file under the marvel data folder

LoadGraph largeHeroes marvel.tsv
GetSize largeHeroes