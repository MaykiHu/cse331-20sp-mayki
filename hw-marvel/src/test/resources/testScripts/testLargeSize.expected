#Tests for the size of a large dataset, the number of nodes, from marvel.tsv
#marvel.tsv should be a valid file under the marvel data folder

loaded graph largeHeroes
size of largeHeroes is 6445